clustering/infinispan/extension/src/main/resources/schema/jboss-as-infinispan_10_0.xsd

```diff
@@ -0,0 +1,1467 @@
+                    </xs:annotation>
+                </xs:attribute>
+            </xs:extension>
+        </xs:complexContent>
+    </xs:complexType>
+
+    <xs:complexType name="scattered-cache">
+        <xs:complexContent>
+            <xs:extension base="tns:segmented-cache">
+                <xs:attribute name="bias-lifespan" type="xs:long" default="300000">
+                    <xs:annotation>
+                        <xs:documentation>When greater than zero, specifies the duration (in ms) that a cache entry will be cached on a non-owner following a write operation.</xs:documentation>
+                    </xs:annotation>
+                </xs:attribute>
+                <xs:attribute name="invalidation-batch-size" type="xs:int" default="128">
+                    <xs:annotation>
+                        <xs:documentation>The threshold after which batched invalidations are sent.</xs:documentation>
+                    </xs:annotation>
+                </xs:attribute>
+            </xs:extension>
+        </xs:complexContent>
+    </xs:complexType>
+
+    <xs:complexType name="store" abstract="true">
+        <xs:sequence>
+            <xs:element name="write-behind" type="tns:write-behind" minOccurs="0" maxOccurs="1">
+                <xs:annotation>
+                    <xs:documentation>Configures a cache store as write-behind instead of write-through.</xs:documentation>
+                </xs:annotation>
+            </xs:element>
+            <xs:element name="property" type="tns:property" minOccurs="0" maxOccurs="unbounded">
+                <xs:annotation>
+                    <xs:documentation>A cache store property with name and value.</xs:documentation>
+                </xs:annotation>
+            </xs:element>
+        </xs:sequence>
+        <xs:attribute name="max-batch-size" type="xs:int" default="100">
+            <xs:annotation>
+                <xs:documentation>
+                    The maximum size of a batch to be inserted/deleted from the store.
+                    If the value is less than one, then no upper limit is placed on the number of operations in a batch.
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="shared" type="xs:boolean" default="false">
+            <xs:annotation>
+                <xs:documentation>This setting should be set to true when multiple cache instances share the same cache store (e.g., multiple nodes in a cluster using a JDBC-based CacheStore pointing to the same, shared database.) Setting this to true avoids multiple cache instances writing the same modification multiple times. If enabled, only the node where the modification originated will write to the cache store. If disabled, each individual cache reacts to a potential remote update by storing the data to the cache store.</xs:documentation>
```

Comment:
Unrelated to this PR I would prefer if we could break this >550 char line. Its not rendered too well on GitHub.